src/components/queryBuilder/utils.ts

```diff
@@ -229,11 +230,15 @@ export const getSQLFromQueryOptions = (options: SqlBuilderOptions): string => {
         options.timeField,
         options.timeFieldType
       );
+      const trendFilters = getFilters(options.filters || []);
+
       if (isDateType(options.timeFieldType)) {
```

Comment:
Shouldn't we throw? Is there a valid case to end up in this branch without `options.timeFieldType` to be set?

Reply:
You are right. We should throw if `options.timeFieldType` isn't valid